test/code-cache/test-code-cache.js

```diff
@@ -26,29 +37,23 @@ const loadedModules = process.moduleLoadList
 // The binary is not configured with code cache, verifies that the builtins
 // are all compiled without cache and we are doing the bookkeeping right.
 if (process.config.variables.node_code_cache_path === undefined) {
-  assert.deepStrictEqual(compiledWithCache, []);
-  assert.notStrictEqual(compiledWithoutCache.length, 0);
-
-  for (const key of loadedModules) {
-    assert(compiledWithoutCache.includes(key),
-           `"${key}" should not have been compiled with code cache`);
-  }
-
+  console.log('The binary is not configured with code cache');
+  assert.deepStrictEqual(compiledWithCache, new Set());
+  assert.deepStrictEqual(compiledWithoutCache, new Set(loadedModules));
 } else {
-  // The binary is configured with code cache.
+  console.log('The binary is configured with code cache');
```

Comment:
I prefer to see this when I run the tests by hand, just so that I don't miss anything because this test is very fail-safe. Do you have a particular reason why we can't console log here?